packages/create-liveblocks-app/bin/templates/init/config-generation.ts

```diff
@@ -96,10 +98,14 @@ export type ThreadMetadata = {
 `;
 }
 
-function reactExports({ framework, suspense, typescript }: InitQuestions) {
+function reactRoomContextExports({
+  framework,
+  suspense,
+  typescript,
+}: InitQuestions) {
   if (framework !== "react") {
     if (typescript) {
-      return `export type TypedRoom = Room<Presence, Storage, UserMeta, RoomEvent, ThreadMetadata>;`;
+      return `export type TypedRoom = Room<Presence, Storage, UserMeta, RoomEvent>;`;
```

Comment:
Removing `ThreadMetadata` from the type argument because `Room` doesn't yet accept the `ThreadMetadata` type argument. See this todo:

https://github.com/liveblocks/liveblocks/blob/main/packages/liveblocks-core/src/room.ts#L464

```ts
export type Room<
  TPresence extends JsonObject,
  TStorage extends LsonObject,
  TUserMeta extends BaseUserMeta,
  TRoomEvent extends Json,
> = CommentsApi<any /* TODO: Remove this any by adding a proper thread metadata on the Room type */> & {
```